test/journeys/lib/test-users.js

```diff
@@ -0,0 +1,37 @@
+import '@ciscospark/plugin-authorization';
+import '@ciscospark/plugin-people';
+import CiscoSpark from '@ciscospark/spark-core';
+import {createUser} from '@ciscospark/test-helper-appid';
+
+import {deconstructHydraId} from './hydra';
+
+/**
+ * Creates and sets up a test user via jwt
+ * @param {object} options
+ * @param {string} options.displayName
+ * @returns {Promise<object>}
+ */
+export default function setupTestUserJwt({displayName}) {
+  return createUser({displayName})
+    .then(({jwt}) => {
+      const guestSpark = new CiscoSpark();
+      return guestSpark.authorization.requestAccessTokenFromJwt({jwt}).then(() =>
+        // We don't have a user id for guest users until a record is looked up
+        guestSpark.people.get('me').then((p) => {
+          const guestUser = p;
```

Comment:
Bypasses the eslint rule, but doesn't actually address the issue. We should use `Object.assign({}, p);` to copy the data instead of referencing it differently.